CONTRIBUTING.md

```diff
@@ -295,75 +294,91 @@ When you are happy with your changes, you can cleanup your environment.
+Open in VS Code with Dev Container extension or use GitHub Codespaces.
+All dependencies and tools are already installed
 
-> [!INFO]
-> Some [dev containers are available](dev-containers) for remote development with docker and pipeline debugging in isolated environments.
-> If you don't want to use one of the dev containers, you can develop the frontend components directly on your local machine.
-> To do so, proceed with installing the necessary dependencies.
+**Or install locally** ([requirements](#dev-requirements)):
+```bash
+pnpm install
+pnpm add -g nx
+```
 
-We use **pnpm** as package manager and **Turbo** for build orchestration. Use angular-eslint/Prettier for linting/formatting.
-VSCode users, check out [this ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [this Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to fix lint and formatting issues during development.
+**Start developing***
+```bash
+nx run @zitadel/login:dev # or console:dev or docs:dev
+```
 
-The commands in this section are tested against the following software versions:
+### Project Overview
 
-- [Docker version 20.10.17](https://docs.docker.com/engine/install/)
-- [Node version v20.x](https://nodejs.org/en/download/)
-- [pnpm version 9.x](https://pnpm.io/installation)
+Choose your contribution area:
 
-To run tests with Cypress, ensure you have installed the required [Cypress runtime dependencies](https://docs.cypress.io/guides/continuous-integration/introduction#Dependencies)
+- **[Login App](#contribute-login)** (Next.js/React) - Modern authentication flows
+- **[Console](#contribute-console)** (Angular) - Admin dashboard and user management  
+- **[Docs](#contribute-docs)** (Docusaurus) - Project documentation
+- **[Packages](#frontend-packages)** - Shared libraries for API communication
 
-<details>
-  <summary>Note for WSL2 on Windows 10</summary>
-  Following the suggestions <a href="https://stackoverflow.com/questions/62641553/setup-cypress-on-wsl-ubuntu-for-windows-10">here </a> subsequently <a href="https://github.com/microsoft/WSL/issues/4106">here </a> may  need to XLaunch and configure your DISPLAY variable. Use at your own risk.
+### Essential Nx Commands
 
-1. Install `VcXsrv Windows X Server`
-2. Set the target of your shortcut to `"C:\Program Files\VcXsrv\xlaunch.exe" -ac`
-3. In WSL2 run `export DISPLAY=$(cat /etc/resolv.conf | grep nameserver | awk '{print $2}'):0` to set your DISPLAY variable
-4. When starting XLaunch, make sure to disable access control
-</details>
+| Task | Command | Notes |
+|------|---------|--------|
+| **Run** | `nx run PROJECT:start` | Production server |
+| **Develop** | `nx run PROJECT:dev` | Hot reload development server |
+| **Test** | `nx run PROJECT:test` | Run all tests |
+| **Lint** | `nx run PROJECT:lint` | Check code style |
+| **Lint Fix** | `nx run PROJECT:lint-fix` | Auto-fix style issues |
```

Comment:
Would it not be better to have a TL;DR with these commands in the beginning of the doc?

This would help fastrack people who want to build Zitadel

Reply:
true, done